charts/sn-platform/templates/streamnative-console/streamnative-console-initialize.yaml

```diff
@@ -17,11 +17,16 @@
 # under the License.
 #
 
-{{- if .Values.initialize }}
 {{- if .Values.components.streamnative_console }}
 apiVersion: batch/v1
 kind: Job
 metadata:
+  annotations:
+    # This is what defines this resource as a hook. Without this line, the
+    # job is considered part of the release.
+    "helm.sh/hook": post-install,post-upgrade
+    "helm.sh/hook-weight": "5"
```

Comment:
A comment on why the hook weight is this value can be useful albeit not necessary. (ex. "# Run after FooBar")

Reply:
thanks @jdmaguire , in fact, it's not necessary here, I will remove it.